Cargo.lock

```diff
@@ -12337,7 +12337,7 @@ dependencies = [
 
 [[package]]
 name = "ic-pub-key"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Did you also mean to bump the version in [ic-pub-key's Cargo.toml](https://github.com/dfinity/ic/blob/master/packages/ic-pub-key/Cargo.toml#L3C12-L3C17) (and for consistency in [BUILD.bazel](https://github.com/dfinity/ic/blob/master/packages/ic-pub-key/BUILD.bazel#L14))?

Reply:
np